assets/js/translation-events.js

```diff
@@ -50,7 +50,29 @@ jQuery(document).ready(function($) {
         });
     });
 
-    function validateEventDates() {
+		$('.delete-event').on('click', function(e) {
+			e.preventDefault();
+			if ( ! confirm( 'Are you sure you want to delete this event?' ) ) {
+				return;
+			}
+			var $form = $('.translation-event-form');
+			$('#form-name').val('delete_event');
+			$('#event-form-action').val( 'delete' );
+			$.ajax({
+				type: 'POST',
+				url: $translation_event.url,
+				data:$form.serialize(),
+				success: function(response) {
+					if ( response.data.eventId ) {
+						history.back();
```

Comment:
After creating an event,  and I'm directed to the edit page, the `Delete` button doesn't show until the edit page is reloaded. We could try to add it in JS.

Reply:
I have solved this problem in [this commit](https://github.com/WordPress/wporg-gp-translation-events/pull/69/commits/22ed7167dc41a7ea7285d00dd17afc1fdeef3f95).